src/redux/features/products.js

```diff
@@ -1,15 +1,67 @@
-import { normalizedProducts } from '../../fixtures';
+import {
+  idle,
+  pending,
+  fulfilled,
+  rejected,
+  REQUEST,
+  SUCCESS,
+  FAILURE,
+} from '../constants';
+import api from '../../api';
 import { arrToMap } from '../utils';
+import produce from 'immer';
 
-export default (state = arrToMap(normalizedProducts), action) => {
-  const { type } = action;
+const LOAD_PRODUCTS = 'LOAD_PRODUCTS';
 
+export const loadProducts = (restId) => ({
+  type: LOAD_PRODUCTS,
+  meta: {
+    apiCall: () => api.loadProducts(restId),
+  },
+  restId,
+});
+
+const initialState = {
+  status: idle,
+  entities: {
+    restLoadingId: [],
+    content: {},
+  },
```

Comment:
можно и без этой вложенности сделать, так будет легче работать с такими данными:
```suggestion
  entities: {},
  restLoadingId: [],
```